packages/jest-circus/src/format_node_assert_errors.js

```diff
@@ -34,31 +35,40 @@ const assertOperatorsMap = {
 const humanReadableOperators = {
   deepEqual: 'to deeply equal',
   deepStrictEqual: 'to deeply and strictly equal',
+  equal: 'to be equal',
   notDeepEqual: 'not to deeply equal',
   notDeepStrictEqual: 'not to deeply and strictly equal',
+  notEqual: 'to not be equal',
+  notStrictEqual: 'not be strictly equal',
+  strictEqual: 'to strictly be equal',
 };
 
 export default (event: Event, state: State) => {
   switch (event.name) {
     case 'test_done': {
-      let assert;
-      try {
-        // Use indirect require so that Metro Bundler does not attempt to
-        // bundle `assert`, which does not exist in React Native.
-        // eslint-disable-next-line no-useless-call
-        assert = require.call(null, 'assert');
-        event.test.errors = event.test.errors.map(error => {
-          return error instanceof assert.AssertionError
-            ? assertionErrorMessage(error, {expand: state.expand})
-            : error;
-        });
-      } catch (error) {
-        // We are running somewhere where `assert` isn't available, like a
-        // browser or React Native. Since assert isn't available, presumably
-        // none of the errors we get through this event listener will be
-        // `AssertionError`s, so we don't need to do anything.
-        break;
-      }
+      event.test.errors = event.test.errors.map(errors => {
+        let error;
+        if (Array.isArray(errors)) {
```

Comment:
this part is really hairy (even though I removed the react native hack).

Ideas welcome!